### PR TITLE
Adding description about PVs used

### PR DESCRIPTION
**Pull request by bjoydeep**
Signed-off-by: Joydeep Banerjee <jbanerje@redhat.com>

Will work off comments entered here by @clyang82 @marcolan018 @dockerymick  in https://github.com/open-cluster-management/multicluster-observability-operator/pull/574

### CODE REVIEW

File: docs/MoreAboutPersistentStorage.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| thanos-receive-default | Thanos receiver accepts incoming data (Prometheus remote-write requests) and writes these into a local instance of the Prometheus TSDB. Periodically (every 2 hours), TSDB blocks are uploaded to the object storage for long term storage and compaction. The amount of hours or days of data retained in this stateful set, which acts a local cache was fixed in API Version `observability.open-cluster-management.io/v1beta`. It has been exposed as an API parameter in `observability.open-cluster-management.io/v1beta2`: _RetentionInLocal_ |
+| thanos-store-shard| It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access. It keeps a small amount of information about all remote blocks on local disk and keeps it in sync with the bucket. This data is generally safe to delete across restarts at the cost of increased startup times. |
+| | |
```
Comment (clyang82) [inline docs/MoreAboutPersistentStorage.md:14]: You may delete the redundant table row.

**Comment by marcolan018** [comment]
this version looks good to me.
hold it unmerged, to wait for comments from @clyang82 @dockerymick

File: docs/MoreAboutPersistentStorage.md (hunk 1 above)
Comment (bjoydeep) [reply docs/MoreAboutPersistentStorage.md]: Thanks @clyang82. Done.

**Pull request merged**